presto-docs/src/main/sphinx/release/unsupported.rst

```diff
@@ -0,0 +1,139 @@
+==========================================
+Unsupported Features and Known Limitations
+==========================================
+
+Presto 148t is equivalent to Presto release 0.148, with some additional features. Below
+are documented unsupported features and known limitations.
+
+Unsupported Functionality
+=========================
+
+Some functionality from Presto 0.148 may work but is not officially supported by Teradata.
+
+* The installation method as documented on `prestodb.io <https://prestodb.io/docs/0.141/installation/deployment.html>`_.
+* Web Connector for Tableau
+* The following connectors:
+  * Cassandra Connector
+  * Kafka Connector
+  * Local File Connector
+  * MongoDB Connector
+  * Redis Connector
+* Developing Plugins
+
+SQL/DML/DDL Limitations
+=======================
+
+ * The SQL keyword ``end`` is used as a column name in ``system.runtime.queries``, so in order to query from that column, ``end`` must be wrapped in quotes
+ * ``NATURAL JOIN`` is not supported
+ * ``EXISTS`` and ``EXCEPT`` are not supported
+ * ``LIMIT ALL`` and ``OFFSET`` are not supported
+
+Hive Connector Limitations
+==========================
+
+Teradata supports data stored in the following formats:
+
+ * Text files
+ * ORC
+ * RCFILE
+ * PARQUET
+
+Hive to Presto data type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Presto does not map Hive data types 1-to-1:
+
+ * All integral types are mapped to ``BIGINT``
+ * ``FLOAT`` and ``DOUBLE`` are mapped to ``DOUBLE``
+ * ``STRING`` and ``VARCHAR`` are mapped to ``VARCHAR``
+
+These mappings may be visible if column values are passed to Hive UDFs or through slight
+differences in mathematical operations.
+
+Because ``FLOAT`` values are mapped to ``DOUBLE``, the user may see unexpected results.
```

Comment:
No. Since we're going to release with native aggregates and scalar functions, that's not true anymore.